src/test/java/uk/ac/cam/cl/dtg/isaac/api/EventsFacadeIT.java

```diff
@@ -38,7 +36,7 @@
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
-public class EventsFacadeTest extends IsaacIntegrationTest {
+public class EventsFacadeIT extends IsaacIntegrationTest {
```

Comment:
I'm not sure i *like* the name change, but I suppose `EventsFacadeIntegrationTest` is worse.

Reply:
Nor do I, but [Failsafe expects this](https://maven.apache.org/surefire/maven-failsafe-plugin/examples/inclusion-exclusion.html) by default - it would be possible to change that with a bit of fiddly configuration however. Looks like they do this to avoid overlapping with Surefire, which runs anything matching `*Test.java` by default.